src/mpi/coll/algorithms/recexchalgo/recexchalgo.c

```diff
@@ -272,7 +272,8 @@ int MPII_Recexchalgo_reverse_digits_step2(int rank, int comm_size, int k)
     int i, T, rem, power, step2rank, step2_reverse_rank = 0;
     int pofk = 1, log_pofk = 0;
     int *digit, *digit_reverse;
-    int mpi_errno = MPI_SUCCESS;
+    int mpi_errno ATTRIBUTE((unused));
+    mpi_errno = MPI_SUCCESS;
```

Comment:
nitpick: none of the other instances in this commit split the assignment onto a second line

Reply:
Fixed. I learned that I could do that somewhere along the way and didn't backport my simplification.